examples/simple/main.tf

```diff
@@ -1,3 +1,8 @@
 # ---------------------------------------------------------------------------------------------------------------------
 # BASIC TERRAFORM CONTENT EXAMPLE
 # ---------------------------------------------------------------------------------------------------------------------
+
+module "aws_ou_scp" {
+  # default_allow = false
+  add_deny_root_account = true
+}
```

Comment:
You might just want to get rid of this. Automated testing for organization modules is pretty hard, since it requires doing something to an actual organization, and most people don't have more than one.